File: digit_ops/digit_ops.swift
/// Вычисляет сумму и произведение цифр числа, а также модуль их разности.
/// 
/// - Parameter num: Натуральное число для обработки

func calculateDigitOps(num: Int) {
    var n = num
    var sum = 0
    var product = 1
    
    while n > 0 {
        let digit = n % 10
        sum += digit
        product *= digit
        n /= 10
    }
    
    print("\(sum) \(product) \(abs(sum - product))")
}

let N = Int(readLine()!)!
calculateDigitOps(num: N)

File: digit_ops/DigitOps.cs
using System;

class DigitOps {
    /// <summary>
    /// Вычисляет сумму и произведение цифр числа, а также модуль их разности.
    /// </summary>
    /// <param name="num">Натуральное число для обработки</param>
    static void CalculateDigitOps(int num) {
        int sum = 0, product = 1;
        int original = num;
        
        while (num > 0) {
            int digit = num % 10;
            sum += digit;
            product *= digit;
            num /= 10;
        }
        
        Console.WriteLine($"{sum} {product} {Math.Abs(sum - product)}");
    }
    
    static void Main() {
        int N = Convert.ToInt32(Console.ReadLine());
        CalculateDigitOps(N);
    }
}